test_model_configs/single_population_node_with_updaters_config.txt

```diff
@@ -0,0 +1,732 @@
+                                    name: "max_age"
+                                    op: LT
+                                    value: "phone"
+                                  }
+                                   sub_filters {
+                                    name: "gender"
+                                    op: EQUAL
+                                    value: "2"
+                                  }
+                                }
+                                node {
+                                  name: "cookied_pool-GB-phone-Demo Bucket - F18-24"
+                                  index: 0
+                                  branch_node {
+                                    branches {
+                                      chance: 0.1930248
+                                      node{
+                                        population_node {
+                                          pools {
+                                            population_offset: 11000000000
+                                            total_population: 30478
+                                          }
+                                        }
+
+                                      }
+                                    }
+                                    branches {
+                                      chance: 0.5157664
+                                      node {
+                                        population_node {
+                                          pools {
+                                            population_offset: 11000030478
+                                            total_population: 81437
+                                          }
+                                        }
+                                      }
+                                    }
+                                    branches {
+                                      chance: 0.24120784999999997
+                                      node {
+                                        population_node {
+                                          pools {
+                                            population_offset: 11000111915
+                                            total_population: 38085
+                                          }
+                                        }
+                                      }
+                                    }
+                                  }
+                                }
+                              }
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+            }
+          }
+          updates {
+            updates{
+              conditional_merge {
+                nodes {
+                   condition {
+                      op: AND
+                      sub_filters {
+                         name: "phone_user_info.demo.gender"
```

Comment:
The top level proto is LabelerEvent. Use labeler_input.profile_info.phone_user_info.demo.gender

Reply:
Added 'labeler_input.profile_info. to all of the sub filters'